make.py

```diff
@@ -125,29 +125,41 @@ def make_bilingual_book(self):
         new_book.metadata = self.origin_book.metadata
         new_book.spine = self.origin_book.spine
         new_book.toc = self.origin_book.toc
-        all_items = list(self.origin_book.get_items())
-        # we just translate tag p
-        all_p_length = sum(
-            [len(bs(i.content, "html.parser").findAll("p")) for i in all_items]
-        )
-        print("TODO need process bar here: " + str(all_p_length))
+        
+        # we just translate tag 
+        with mp.Pool() as pool:
+            translated_p_list = pool.map(
+                lambda x: (x[0], self.translate_model.translate(x[1])),
+                [(i, p.string) for i in self.origin_book.get_items()
+                    if i.get_type() == 9
+                    for p in bs(i.content, "html.parser").findAll("p")
+                    if p.string and not p.string.isdigit()]
+                )
```

Comment:
what I am thinking batch is 
<p1> + <p2> + <p3> -> chatgpt api -> <p1_t> <pt_2> <p3_t>
then we split it and find the index then add the `translate_p` tag after.


And your work also  fine, I will merge after review.

Reply:
batch text to chatgpt api also well